generate-adjudicator-bindings.sh

```diff
@@ -10,7 +11,9 @@ abigen --abi=$(pwd)/tmp-build/ConsensusApp.abi --bin=$(pwd)/tmp-build/ConsensusA
 # Token
 abigen --abi=$(pwd)/tmp-build/Token.abi --bin=$(pwd)/tmp-build/Token.bin --pkg=Token --type=Token --out=$(pwd)/../client/engine/chainservice/erc20/Token.go
 # VirtualPaymentApp
-abigen --abi=$(pwd)/tmp-build/VirtualPaymentApp.abi --bin=$(pwd)/tmp-build/VirtualPaymentApp.bin --pkg=VirtualPaymentApp --out=$(pwd)/../client/engine/chainservice/virtualpaymentapp/VirtualPaymentApp.go 
+abigen --abi=$(pwd)/tmp-build/VirtualPaymentApp.abi --bin=$(pwd)/tmp-build/VirtualPaymentApp.bin --pkg=VirtualPaymentApp --out=$(pwd)/../client/engine/chainservice/virtualpaymentapp/VirtualPaymentApp.go
+
+abigen --abi=$(pwd)/tmp-build/Create2Deployer.abi --bin=$(pwd)/tmp-build/Create2Deployer.bin --pkg=Create2Deployer --out=$(pwd)/../client/engine/chainservice/create2deployer/Create2Deployer.go
```

Comment:
```suggestion
# Create2 Deployer
abigen --abi=$(pwd)/tmp-build/Create2Deployer.abi --bin=$(pwd)/tmp-build/Create2Deployer.bin --pkg=Create2Deployer --out=$(pwd)/../client/engine/chainservice/create2deployer/Create2Deployer.go
```

Reply:
Good catch. This script is refactored in https://github.com/statechannels/go-nitro/pull/921 though.